_posts/2020-10-21-djangocon-us-2020-video.md

```diff
@@ -0,0 +1,24 @@
+---
+author: Katia Lira
+category: General
+date: 2020-10-21 11:00:00
+layout: post
+image: /static/img/blog/announcing-2019.png
```

Comment:
Is this the right image that we want?

(Just double-checking)

Reply:
![image](https://user-images.githubusercontent.com/7773256/96750168-41335180-1391-11eb-8690-3c37a6a33397.png)

Update: looks like the image doesn't render, so I guess it's okay?

Reply:
Update 2: it appears in slack previews, so we don't want that image.

![image](https://user-images.githubusercontent.com/7773256/96750260-5e682000-1391-11eb-9a3e-efc1f65a053e.png)

Reply:
Fixed!